packages/Pillow/patches/0001-Enable-image-formats.patch

```diff
@@ -1,14 +1,14 @@
-From 0c4d089d29d7b12de69b1725a65ad1d3d3fa82ec Mon Sep 17 00:00:00 2001
+From c5b15988defebfd88bb3812a81179a92219047ad Mon Sep 17 00:00:00 2001
 From: ryanking13 <def6488@gmail.com>
 Date: Tue, 19 Apr 2022 07:18:31 +0000
 Subject: [PATCH] Enable image formats
 
 ---
- setup.py | 19 ++++++++++++++-----
- 1 file changed, 14 insertions(+), 5 deletions(-)
+ setup.py | 23 +++++++++++++++++------
+ 1 file changed, 17 insertions(+), 6 deletions(-)
 
 diff --git a/setup.py b/setup.py
```

Comment:
It looks like Pillow already provides some ways to set CFLAGS / LDFLAGS using env variables. So maybe we can remove this patch entirely with some effort. The issue is that it is a bit uncomfortable to get the path of library ports that emscripten provides (e.g. `libjpeg`).

I'll see if we can build them ourselves using `embuilder` (https://github.com/emscripten-core/emscripten/issues/9353#issuecomment-526683501) so that we can install them into `WASM_LIBRARY_DIR`.